Source/Core/VideoCommon/VideoBackendBase.h

```diff
@@ -31,34 +31,34 @@ enum EFBAccessType
 struct SCPFifoStruct
 {
 	// fifo registers
-	volatile u32 CPBase;
-	volatile u32 CPEnd;
+	u32 CPBase;
+	u32 CPEnd;
 	u32 CPHiWatermark;
 	u32 CPLoWatermark;
-	volatile u32 CPReadWriteDistance;
-	volatile u32 CPWritePointer;
-	volatile u32 CPReadPointer;
-	volatile u32 CPBreakpoint;
-	volatile u32 SafeCPReadPointer;
+	u32 CPReadWriteDistance;
```

Comment:
This one is the main variable to store how much work the GPU still has to do. So it likely needs to be atomic.